lib/msal-common/src/network/ThrottlingUtils.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { NetworkResponse } from "./NetworkManager";
+import { ServerAuthorizationTokenResponse } from "../server/ServerAuthorizationTokenResponse";
+import { Constants, HeaderNames } from "../utils/Constants";
+import { CacheManager } from "../cache/CacheManager";
+import { StringUtils } from "../utils/StringUtils";
+
+export type RequestThumbprint = {
```

Comment:
These types should each be in their own separate files

Reply:
Type only exports end up requiring typescript 3.9 in the client application, so we're trying to avoid that (at least for now).

Reply:
Is that relevant for types that aren't considered publicly facing? We already have many Type only exports across both common and browser

Reply:
Ah, that may not be the case for types that aren't part of the main export.

Reply:
When we say "public" here, do we mean being consumed by browser/node, or actually visible to customers? Because at least browser will need to use these to cover a few cases, I think

Reply:
I ended up putting these in separate files.